test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+FBASE_SNIPPET=test/travis/firebase_init.cc.snip
+FBASE_NOWIFI_SNIPPET=test/travis/no_firebase_init.cc.snip
+for example in `find examples/ -name *.ino`;
+do
+  echo $example;
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_SNIPPET | tr -d '\n'`) ||
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_NOWIFI_SNIPPET | tr -d '\n'`) ||
```

Comment:
Well the `tr -d \n` is actually for removing the formatting of the xxd output. It normally outputs like 80 hex chars per line. I didn't see an option to have it just give me one large hex string. So that isn't effecting the actual source data. We are comparing the exact source bytes between the files, no manipulations there.

Which is why this is technically better (but probably inconsequentially so) than removing the line breaks from the source files as `echo` would do, that could mask differences like these two would match if we did that:

#define WIFI_
SSID 
"SSI
D"

and

#define WIFI_SSID "SSID"